content/en/security/cloud_security_management/vulnerabilities/_index.md

```diff
@@ -30,45 +30,92 @@ further_reading:
+Explore reports
+: View and monitor vulnerability data in your dashboards.
 
-## Triage and remediate
+## Deployment methods
 
-The [Vulnerabilities Explorer][1] also offers triaging options for detected vulnerabilities that enable you to change the status of a vulnerability, and assign it to individual members for remediation and tracking.
+Get started with CSM Vulnerabilities and cover your infrastructure in minutes, using:
+- [Agentless scanning][11]
+- [Unified Datadog agent][12]
 
-**Note**: To help you focus on the vulnerabilities that truly matter, vulnerabilities are auto-closed for infrastructure that is either no longer running, or contains the remediated fixed version of the previously-vulnerable package.
+You can also use both deployment methods to use the unified Datadog Agent where you already have it deployed, and Agentless elsewhere.
 
-{{< img src="security/vulnerabilities/csm_remediate.png" alt="Details explorer of a specific vulnerability highlighting the ability to remediate and assign to team member" width="100%">}}
+After you've enabled it, Datadog starts scanning your resources continuously, and starts reporting prioritized vulnerabilities in your [CSM Vulnerability Explorer][1] within an hour. 
+
+Use these tables to decide which solution to start with: 
+| Feature                                   | Agentless                                     | Unified Datadog Agent          |
+|-------------------------------------------|-----------------------------------------------|--------------------------------|
+| Time to deploy across your infrastructure | Minutes                                       | Hours to weeks                 |
+| Vulnerability prioritization              | Yes                                           | Yes, with runtime context      |
+| Vulnerability scanning frequency          | 12 hours                                      | Real-time                      |
+
+| Vulnerability detection scope             | Agentless                                     | Unified Datadog Agent          |
+|-------------------------------------------|-----------------------------------------------|--------------------------------|
+| Host and host image                       | OS packages and app packages, mapped to image | OS packages                    |
+| Container image                           | OS packages and app packages, mapped to image | OS packages                    |
+| Cloud provider                            | AWS, [Azure (Preview)][15]                    | AWS, Azure, GCP, on-prem, etc. |
+| Operating system                          | Linux                                         | Linux, Windows                 |
+| Serverless                                | AWS Lambda                                    | -                              |
+| Container registries                      | [Amazon ECR (Preview)][16]                    | -                              |
```

Comment:
I'm assuming the dashes indicate a lack of support? I suggest using "Not applicable" or something along those lines.

Reply:
Checking other documentation for consistency, looks like we typically leave an empty space. 
Example: https://docs.datadoghq.com/agent/supported_platforms/?tab=linux